src/main/kotlin/com/faforever/userservice/config/DatabaseConfig.kt

```diff
@@ -0,0 +1,39 @@
+package com.faforever.userservice.config
+
+import io.r2dbc.spi.ConnectionFactory
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.convert.converter.Converter
+import org.springframework.data.convert.CustomConversions
+import org.springframework.data.convert.ReadingConverter
+import org.springframework.data.r2dbc.convert.R2dbcCustomConversions
+import org.springframework.data.r2dbc.dialect.DialectResolver
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+@ReadingConverter
+class LocalDateTimeToOffsetDateTimeReadingConverter : Converter<LocalDateTime, OffsetDateTime> {
+    private val log = LoggerFactory.getLogger(LocalDateTimeToOffsetDateTimeReadingConverter::class.java)
```

Comment:
this should be moved into a companion object, since it's a static thing. Also uppercase because static

Reply:
Done